components/Tabs/src/private/MDCItemBar.m

```diff
@@ -328,6 +328,21 @@ - (void)tintColorDidChange {
   [self updateColors];
 }
 
+// UISemanticContentAttribute was added in iOS SDK 9.0 but is available on devices running earlier
+// version of iOS. We ignore the partial-availability warning that gets thrown on our use of this
+// symbol.
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wpartial-availability"
+- (void)mdf_setSemanticContentAttribute:(UISemanticContentAttribute)semanticContentAttribute {
+  if (semanticContentAttribute == self.mdf_semanticContentAttribute) {
+    return;
+  }
+  [super mdf_setSemanticContentAttribute:semanticContentAttribute];
+  _collectionView.mdf_semanticContentAttribute = semanticContentAttribute;
+  [_collectionView.collectionViewLayout invalidateLayout];
```

Comment:
Can we skip calling `invalidateLayout` if the semantic content attribute value hasn't changed?

Reply:
Ah, sorry. I somehow missed that completely. :-\